gramex/handlers/mlhandler.py

```diff
@@ -241,13 +241,26 @@ def get(self, *path_args, **path_kwargs):
         else:
             self._check_model_path()
             if '_download' in self.args:
-                self.set_header('Content-Type', 'application/octet-strem')
+                self.set_header('Content-Type', 'application/octet-stream')
                 self.set_header('Content-Disposition',
                                 f'attachment; filename={op.basename(self.store.model_path)}')
                 with open(self.store.model_path, 'rb') as fout:
                     self.write(fout.read())
             elif '_model' in self.args:
                 self.write(json.dumps(self.model.get_params(), indent=2))
+            elif len(self.request.files.keys()) and \
+                self.request.files['image'][0].content_type in \
+                    ['image/jpeg', 'image/jpg', 'image/png']:
```

Comment:
This check should not happen here. As far as possible, send the request payload blindly to the `_fit` or `_predict` methods. The wrapper class should take care of everything if it is written correctly.

Also note that there are two ways one can send images in a request.

1. Send files (multipart form data) - in which case you look at the mimetypes of the files received and open them accordingly. For this, take a look at the definition of `self._parse_multipart_form_data`
2. Send the raw bytestream of an image with a `Content-Type: image/whatever` header. In this case, write functions called `_parse_image_jpeg`, `_parse_image_png`, etc. The handler will automatically call them.

Basically the handler knows how to parse data given the content type of the request.